packages/dso-toolkit/src/components/legend-item/legend-item.scss

```diff
@@ -0,0 +1,12 @@
+@use "legend-item.variables" as legend-item-variables;
+@use "../../global/mixins/symbol.mixin" as symbol;
+
+dso-legend-item {
```

Comment:
Afgelopen maandag hebben we het over de `@mixin children()` constructie (en haar gebreken) gehad, en gaan we ons best doen om dit niet meer te doen.

Omdat dit werk al in omloop was vind ik het ook goed als we het voor deze change laten, en daarom stip ik dit stukje code alleen ter bewustwording aan.